### ISSUE TITLE
Uncaught TypeError: Cannot set property 'en' of undefined en.js:1

### ISSUE THREAD

**Issue by TheKojuEffect** [jhipster/generator-jhipster#3264]
##### **Overview of the issue**

Error messages in console 
```
Uncaught TypeError: Cannot set property 'en' of undefined    en.js:1
Uncaught TypeError: Cannot set property 'fr' of undefined    fr.js:1
```

##### **JHipster Version(s)** 

3.0.0

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder** 
```
{
  "generator-jhipster": {
    "jhipsterVersion": "3.0.0",
    "baseName": "hipster",
    "packageName": "io.example.hipster",
    "packageFolder": "io/example/hipster",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": "no",
    "websocket": "no",
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "prodDatabaseType": "postgresql",
    "searchEngine": "no",
    "buildTool": "gradle",
    "enableSocialSignIn": false,
    "jwtSecretKey": "6b7351dc7535683df733688a4c69v0563e90178w",
    "useSass": false,
    "applicationType": "monolith",
    "testFrameworks": [],
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en",
      "fr"
    ]
  }
}
```

##### **Browsers and Operating System** 

Chrome

##### **Suggest a Fix** 

It seems like it is unnecessary to load `lang.js` (i.e `en.js`, `fr.js`) in `index.html`. If I don't import those files, the error messages doesn't show up and no problem with internationalization.

**Comment by andidev** [comment]
Not sure why you get this error. Have you updated your versions in bower.json? Or done any other changes?

Exactly when do you get that error, on first load of the application?

The message format is needed for the pluralization to work. Are you really sure it works without it?

**Comment by TheKojuEffect** [comment]
Hi @andidev, I am getting the error messages in fresh jhipster 3.0.0 application. The messages show up only one time or when I refresh. And the pluralization seems to be working without those `lang.js` files.

**Comment by andidev** [comment]
Okey so it looks like angular-translate-interpolation-messageformat is updated to latest version and therefore the inclusion of the language files are no longer needed. AR least they are not there in the latest message format version. So you are right. I'll look into this later and fix it.

**Comment by andidev** [comment]
I just tried to generate a new project and strangely I do not get your error in the console.
The reason why I don't get the error is since the messageformat dependency that is downloaded by bower (transitively) is a version that does not contain the lang.js files that you mention. And therefore the bower overrides including the lang.js files are ignored.

Are you still having this issue if you do a fresh install of a jhipster 3.0 app?

Since the lang.js files are no longer needed we can remove method [addMessageformatLocaleToBowerOverride](https://github.com/jhipster/generator-jhipster/blob/ee19bf8bb599c053d4b4f69becd4ca33a16d67bb/generators/generator-base.js#L40).

And also maybe we should add messageformat to bower.json as a dependency and set a fix version in it since otherwise it will be included transitively by angular-translate-interpolation-messageformat set to a non fix version "~0.3.0".

**Comment by deepu105** [comment]
@andidev yes I think we should set an explicit version
